pages/your_tasks/data_security.md

```diff
@@ -18,12 +19,14 @@ dsw:
     the data?
   uuid: a30f5047-33c1-45a7-8b3f-b1b90c364fc9
 faircookbook:
-- name: Licensing Data
```

Comment:
faircookbook changes need to be applied to this file: https://github.com/elixir-europe/faircookbook-rdmkit/blob/main/faircookbook_rdmkit_mapping.yml

Reply:
changes are prepared here: https://github.com/elixir-europe/faircookbook-rdmkit/pull/41